impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -403,30 +430,33 @@ static bool EncodeCommand(const Context& context,
       command.stencil_reference);
 
   // Configure vertex and index and buffers for binding.
-  auto& vertex_buffer_view = command.vertex_buffer.vertex_buffer;
-
-  if (!vertex_buffer_view) {
-    return false;
-  }
+  std::vector<vk::Buffer> vertex_buffers;
+  std::vector<vk::DeviceSize> vertex_buffer_offsets;
```

Comment:
Move these under the else if clause so we don't have per-command heap allocation.